core/src/Revolution/Mail/modPHPMailer.php

```diff
@@ -222,13 +222,14 @@ public function send(array $attributes = [])
 
         $sent = false;
         try {
-            if (strpos($this->mailer->ContentType, 'html') !== false) {
-                if (!empty($this->mailer->Body)) {
-                    $html = new InlineStyle($this->mailer->Body);
-                    /** @noinspection PhpParamsInspection */
-                    $html->applyStylesheet($html->extractStylesheets());
-                    $this->mailer->Body = $html->getHTML();
-                }
+            if (!empty($this->mailer->Body) && (strpos($this->mailer->ContentType, 'html') !== false)) {
+                $body = $this->mailer->Body;
+                // Turn UTF-8 characters into entities
+                $body = mb_convert_encoding($body, 'HTML-ENTITIES', "UTF-8");
```

Comment:
Can you make this conditional based on the existence of the mbstring extension? Otherwise, this adds a non-default extension to the product requirements and would also need to be added to composer.json.

Reply:
How do you feel about including the symfony m string polyfill instead? Pretty lightweight, would allow simplifying some code.

Reply:
Oh hey, fun fact, we're already including `symfony/polyfill-mbstring`:

```
composer why symfony/polyfill-mbstring
mtdowling/jmespath.php  2.6.1   requires  symfony/polyfill-mbstring (^1.17)  
symfony/console         v5.4.7  requires  symfony/polyfill-mbstring (~1.0)   
symfony/string          v5.4.3  requires  symfony/polyfill-mbstring (~1.0)  
```

Would be nice to make it explicit in the composer.json, but looks like there's no need for function checks on mbstring functions anymore.